input.go

```diff
@@ -234,6 +234,15 @@ func IsStandardGamepadLayoutAvailable(id GamepadID) bool {
 	return uiDriver().Input().IsStandardGamepadLayoutAvailable(id)
 }
 
+// UpdateStandardGamepadLayoutMappings updates the gamepad layout definitions using a set of definitions in SDL_GameControllerDB format.
+//
+// This functionality is only supported on desktop systems and will just return an error elsewhere.
+//
+// UpdateStandardGamepadLayoutMappings must be called on the main thread before the main loop.
```

Comment:
Is this true? We should be able to make this concurrent-safe.

Reply:
Done now (by moving it to the ui.t thread as many other places do).

This doesn't seem to be possible BEFORE the main loop is launched, so before the main loop it remains concurrent-unsafe.